index.html

```diff
@@ -0,0 +1,1447 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+
+    <!-- CSS file -->
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+
+    <title>Newsweek</title>
+  </head>
+
+
+  <body>
+
+<div class="header-screen d-flex">
+        
+<div class="header-left no-big">
+<img src="images/header.png">
+</div>
+
+
+<div class="header-right d-flex no-big">
+
+<div class="header-right-l">
+</div>
+<div class="header-right-r d-flex justify-content-center">
+        <h2 class="signin-h no-big">SIGN IN</h2>
+        <h2 class="subscribe-h no-big">SUBSCRIBE</h2>
+        <div class="bars-h no-big"><i class="fas fa-bars"></i></div>
+</div>
+
+</div>
+</div>
+
+    <div class = "container-fluid first">
+  <header class="text-center">
+<div class="date-h no-small">Thu, May 23, 2019 </div>
+        <img class="header-img no-small" alt="header" src="images/header.png">
+
+        <div class="btn-h-big no-small d-flex">
+                <h2 class="signbh no-small">SIGN UP</h2><h2 class="subsbh no-small">SUBSCRIBE ></h2>
+        </div>
+
+
+
+  </header>
+
+  <navbar class="navbar navbar-expand-md no-small">
+
+                <a class="navbar-brand" href="#">U.S.</a>
+                <a class="navbar-brand" href="#">World</a>
+                <a class="navbar-brand" href="#">Business</a>
+                <a class="navbar-brand" href="#">Tech & Science</a>
+                <a class="navbar-brand" href="#">Culture</a>
+                <a class="navbar-brand" href="#">Sports</a>
+                <a class="navbar-brand" href="#">Health</a>
+                <a class="navbar-brand" href="#">Opinion</a>
+                <a class="navbar-brand" href="#">Ndelible</a>
+
+                <form class="form-inline my-2 my-lg-0">
+                                <input class="form-control mr-sm-2" type="search" placeholder="Search" aria-label="Search">
+                                <button class="btn btn-light my-2 my-sm-0" type="submit"><i class="fas fa-search"></i></button>
+                              </form>
+
+
+</navbar>
+    </div>
+
+
+<!-- ***** STARTS 2TH COLUMN ****** --->
```

Comment:
extra - in your comment (not necessary to remove but recommended

Reply:
corrected in the last commit